### ISSUE TITLE
panic when relay gets disconnected

### ISSUE THREAD

**Issue by jb55** [thesimplekid/cln-nostr-connect#4]
```
thread 'thread 'thread 'mainmainmain' panicked at '' panicked at '' panicked at 'Error reading message: Protocol(ResetWithoutClosingHandshake)Error
 reading message: Protocol(ResetWithoutClosingHandshake)Error reading message: Protocol(ResetWithoutClosingHandshake)', thread '', ', mainsrc/main.
rssrc/main.rs' panicked at 'Error reading message: Protocol(ResetWithoutClosingHandshake)', src/main.rs:386:::386386::18src/main.rs18
thread '18                                                                                                                                         
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace                                                        
:386:                                                                                                                                              
18                                                                                                                                                 
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace                                                        
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace                                                                
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace                                                                      main' panicked at 'Error reading message: Protocol(ResetWithoutClosingHandshake)', src/main.rs:386:18
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace                                                                
2023-05-16T18:07:00.888Z INFO    plugin-cln-nostr-wallet-connect: Disconnected from ws://127.0.0.1:8080/                                           thread 'main' panicked at 'Error reading message: Protocol(ResetWithoutClosingHandshake)', src/main.rs:424:18
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace                                                        
thread 'thread 'thread 'mainmainmain' panicked at '' panicked at '' panicked at 'Error reading message: Protocol(ResetWithoutClosingHandshake)Error
 reading message: Protocol(ResetWithoutClosingHandshake)Error reading message: Protocol(ResetWithoutClosingHandshake)', ', ', src/main.rssrc/main.r
ssrc/main.rs:::376376376:::1818                                                                                                                    
18                                                                                                                                                 
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace                                                        
                                                                                                                                                   
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace 
```

**Comment by jb55** [comment]
also just seems to die and not restart or anything

**Comment by thesimplekid** [comment]
been meaning to fix this for awhile. Should be fixed by a776e2e80ba4f439622f59bbfbc84081d292d3e5 with a small fix for my delay logic in 00439c38e54a60c0e7262db20183686d41488349
